ansible_collections/arista/avd/roles/eos_designs/docs/tables/node-type-wan-configuration.md

```diff
@@ -0,0 +1,85 @@
+    | [<samp>&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;group</samp>](## "<node_type_keys.key>.node_groups.[].group") | String | Required, Unique |  |  | The Node Group Name is used for MLAG domain unless set with 'mlag_domain_id'.<br>The Node Group Name is also used for peer description on downstream switches' uplinks.<br> |
+    | [<samp>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;nodes</samp>](## "<node_type_keys.key>.node_groups.[].nodes") | List, items: Dictionary |  |  |  | Define variables per node. |
+    | [<samp>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;name</samp>](## "<node_type_keys.key>.node_groups.[].nodes.[].name") | String | Required, Unique |  |  | The Node Name is used as "hostname". |
+    | [<samp>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;sdwan_site</samp>](## "<node_type_keys.key>.node_groups.[].nodes.[].sdwan_site") | String |  |  |  |  |
+    | [<samp>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;dps_mss_ipv4</samp>](## "<node_type_keys.key>.node_groups.[].nodes.[].dps_mss_ipv4") | Integer |  |  | Min: 64<br>Max: 65495 | Segment Size for IPv4. |
+    | [<samp>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;dps_mss_ipv6</samp>](## "<node_type_keys.key>.node_groups.[].nodes.[].dps_mss_ipv6") | Integer |  |  | Min: 64<br>Max: 65475 | Segment Size for IPv6. |
+    | [<samp>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;sdwan_site</samp>](## "<node_type_keys.key>.node_groups.[].sdwan_site") | String |  |  |  |  |
+    | [<samp>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;dps_mss_ipv4</samp>](## "<node_type_keys.key>.node_groups.[].dps_mss_ipv4") | Integer |  |  | Min: 64<br>Max: 65495 | Segment Size for IPv4. |
+    | [<samp>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;dps_mss_ipv6</samp>](## "<node_type_keys.key>.node_groups.[].dps_mss_ipv6") | Integer |  |  | Min: 64<br>Max: 65475 | Segment Size for IPv6. |
+    | [<samp>&nbsp;&nbsp;nodes</samp>](## "<node_type_keys.key>.nodes") | List, items: Dictionary |  |  |  | Define variables per node. |
+    | [<samp>&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;name</samp>](## "<node_type_keys.key>.nodes.[].name") | String | Required, Unique |  |  | The Node Name is used as "hostname". |
+    | [<samp>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;sdwan_site</samp>](## "<node_type_keys.key>.nodes.[].sdwan_site") | String |  |  |  |  |
+    | [<samp>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;dps_mss_ipv4</samp>](## "<node_type_keys.key>.nodes.[].dps_mss_ipv4") | Integer |  |  | Min: 64<br>Max: 65495 | Segment Size for IPv4. |
+    | [<samp>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;dps_mss_ipv6</samp>](## "<node_type_keys.key>.nodes.[].dps_mss_ipv6") | Integer |  |  | Min: 64<br>Max: 65475 | Segment Size for IPv6. |
+
+=== "YAML"
+
+    ```yaml
+    <node_type_keys.key>:
+
+      # Define variables for all nodes of this type.
+      defaults:
+        sdwan_site: <str>
```

Comment:
Would it make sense to move the role attribute (currently in node type) to be a site level thing?